index.html

```diff
@@ -20,6 +20,7 @@ <h1 class="nav_brand">Rapheal Okonta</h1>
         <li class="nav_link"><a href="#">Portfolio</a></li>
         <li class="nav_link"><a href="#">About</a></li>
         <li class="nav_link"><a href="#">Contact</a></li>
+        <li class="nav_link_email"><a href="#"><img src="image folder/Disabled.png" alt=""></a></li>
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible. your logo and content should be inline on the same line and also increase the width of the `p` tag so it matches the design. See the image below for reference. The text is aligned to the left but it doesn't pass the middle of the page.
### Your design
![his](https://user-images.githubusercontent.com/81727988/206657205-0974fb90-fb57-40b0-a528-20b18dd611a1.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/206657242-10e1755b-abb6-4a3e-b281-4ef70ea1fc8c.png)